examples/nlp/intent_slot_classification/conf/intent_slot_classification_config.yaml

```diff
@@ -0,0 +1,91 @@
+# Sentence classification with pretrained BERT models
+
+pl:
+  trainer:
+    gpus: 1 # the number of gpus, 0 for CPU
+    num_nodes: 1
+    max_epochs: 100
+    max_steps: null # precedence over max_epochs
+    accumulate_grad_batches: 1 # accumulates grads every k batches
+    amp_level: O0 # O1/O2 for mixed precision
+    distributed_backend: ddp
+    checkpoint_callback: False  # Provided by exp_manager
+    logger: False  # Provided by exp_manager
+
+model:
+  data_dir: ??? # /path/to/data
+  class_balancing: null # or weighted_loss
+  intent_loss_weight: 0.6 # only if class_balancing = weighted_loss
+  pad_label: '-1'
+  none_slot_label: 'O'
+  ignore_extra_tokens: false
+  ignore_start_end: false
+
+  train_ds:
```

Comment:
You may restructure your data loaders like this: https://github.com/NVIDIA/NeMo/pull/910
It moves some params like num_workers, drop_last, pin_memory to a dataset section.

Reply:
Let me do this step afterwards. I want to push working code to Candidates.